string name
int32 state_cur
---
int32 action
